best-practices/nginx_path_checks/restrict-path.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-ingress-paths
+  annotations:
+    policies.kyverno.io/title: Restrict NGINX Ingress path values 
+    policies.kyverno.io/category: Security
+    policies.kyverno.io/severity: high
+    policies.kyverno.io/subject: Ingress, NGINX Ingress
+    policies.kyverno.io/minversion: 1.6.0
+    policies.kyverno.io/description: |-
+      This policy mitigates CVE-2021-25745 by restricting `spec.rules[].http.paths[].path` to safe values.
+spec:
+  rules:
+    - name: check-paths
+      match:
+        resources:
```

Comment:
Let's put this under either an `any` or `all` block to be consistent with conventions.